cristin-import/src/test/java/no/unit/nva/cristin/CristinDataGenerator.java

```diff
@@ -21,60 +38,80 @@
 import no.unit.nva.publication.s3imports.FileContentsEvent;
 import nva.commons.core.JsonUtils;
 
-import java.net.URI;
-import java.time.LocalDate;
-import java.util.ArrayList;
-import java.util.Calendar;
-import java.util.Date;
-import java.util.List;
-import java.util.Random;
-import java.util.Set;
-import java.util.stream.Collectors;
-import java.util.stream.IntStream;
-import java.util.stream.Stream;
-
-import static no.unit.nva.cristin.mapper.CristinObject.MAIN_CATEGORY_FIELD;
-import static no.unit.nva.cristin.mapper.CristinObject.PUBLICATION_OWNER_FIELD;
-import static no.unit.nva.cristin.mapper.CristinObject.SECONDARY_CATEGORY_FIELD;
-import static no.unit.nva.hamcrest.DoesNotHaveEmptyValues.doesNotHaveEmptyValues;
-import static no.unit.nva.hamcrest.DoesNotHaveEmptyValues.doesNotHaveEmptyValuesIgnoringFields;
-import static nva.commons.core.attempt.Try.attempt;
-import static org.hamcrest.MatcherAssert.assertThat;
-
-public class CristinDataGenerator {
+public final class CristinDataGenerator {
```

Comment:
most changes here are that all methods are static now and some have become public as well.,